spec/filters_spec.rb

```diff
@@ -26,15 +26,15 @@
       expect(SimpleCov::StringFilter.new("sample.rb")).to be_matches subject
     end
 
+    it "doesn't match a new SimpleCov::StringFilter '.pl'" do
+      expect(SimpleCov::StringFilter.new(".pl")).not_to be_matches subject
+    end
+
     it "doesn't match a parent directory with a new SimpleCov::StringFilter" do
       parent_dir_name = File.basename(File.expand_path("..", File.dirname(__FILE__)))
       expect(SimpleCov::StringFilter.new(parent_dir_name)).not_to be_matches subject
     end
 
-    it "matches a new SimpleCov::StringFilter '/fixtures/'" do
-      expect(SimpleCov::StringFilter.new("sample.rb")).to be_matches subject
-    end
```

Comment:
why id you remove this spec, shouldn't that still work?

Reply:
Because these are exactly same (though the latter description is wrong):

https://github.com/colszowka/simplecov/blob/c5e8654834e922d3e69a980aef029d4bcc880b6e/spec/filters_spec.rb#L25-L27
https://github.com/colszowka/simplecov/blob/c5e8654834e922d3e69a980aef029d4bcc880b6e/spec/filters_spec.rb#L34-L36

Maybe a mistake of copy-pasting?

Reply:
Ah yes, missed that! Seems like it was meant to be `"/fixtures/"` indeed! Unless I missed that one as well :)